allensdk/brain_observatory/behavior/schemas.py

```diff
@@ -26,8 +26,8 @@ class SubjectMetadataSchema(RaisingSchema):
     # In this case they already exist in the 'Subject' builtin pyNWB class
     neurodata_skip = {"age_in_days", "genotype", "sex", "subject_id"}
 
-    age_in_days = fields.String(
-        doc='Age of the specimen donor/subject (in days)',
+    age = fields.String(
```

Comment:
I would prefer we not have an additional `age` field when we already have pyNWB class `BehaviorSubject` (augmented pyNWB 'Subject') to store age.

Reply:
@njmei it was always like this. The value in `neuroadata_skip` needs to be updated